packages/api/src/index.js

```diff
@@ -50,19 +51,24 @@ router.get('/', () => {
 router.get('/error', () => { throw new Error('A deliberate error!') })
 router.all('*', withCorsHeaders(() => notFound()))
 
-function serverError (request, error) {
+/**
+ * @param {Error} error
+ * @param {Request} request
+ * @param {import('./env').Env} env
+ */
+function serverError (error, request, env) {
   console.error(error.stack)
-  const message = error.message || 'Server Error'
-  const status = error.status || 500
-  return addCorsHeaders(request, new JSONResponse({ message }, { status }))
+  env.sentry && env.sentry.captureException(error)
```

Comment:
Please can this move into the error handler, and only report when the status is >=500...also the `console.error` line?